src/components/elements/favorites-list.tsx

```diff
@@ -0,0 +1,103 @@
+"use client";
+
+import useFavorites from "@lib/hooks/useFavorites";
+import { ChatBubbleLeftEllipsisIcon, ClipboardDocumentIcon, EnvelopeIcon, HeartIcon} from "@heroicons/react/24/outline";
+import {useIsClient} from "usehooks-ts";
+import { XMarkIcon } from "@heroicons/react/20/solid";
+
+
+const ShareButtons = () => {
+  const baseUrl = window.location.origin;
+  const { favs } = useFavorites();
+  const urlParams = `/favorites?fav=${favs.map(fav => `${fav.uuid}:${fav.title}:${fav.units}`).join(",")}`;
```

Comment:
Probably easier to just use the `uuid` only. Do we need to pass along all that params? I could see it getting pretty long.